content/docs/user-guide/experiment-management/visualizing-plots.md

```diff
@@ -160,8 +167,6 @@ the default plotting behavior (apply the `linear` plot [template] to the last
 found column):
 
 ```yaml
-# dvc.yaml
----
```

Comment:
Remove these comments since it's mentioned briefly before what file we're working with.